content/01.abstract.md

```diff
@@ -1,3 +1,7 @@
 ## Abstract {.page_break_before}
 
-
+Scientific literature reviews are ideal for open, collaborative writing.
+Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
+However, the traditional multi-author writing process breaks down at scale.
+We present techniques for overcoming the challenges of open manuscript writing.
```

Comment:
Very strong first four sentences!

Do we want to mention the deep review for context? Something like "based on our experience with the Deep Review, where XX authors combined expertise to review deep learning in precision medicine.

Reply:
My inclination was to keep it as short as possible.  I read several feature article abstracts and some were only 3 sentences.